test/c/test_version_info.c

```diff
@@ -0,0 +1,21 @@
+#include "common.h"
+
+// Test that QISKIT_VERSION_MAJOR, _MINOR, _PATCH are defined and print them
+int test_version_macros() {
```

Comment:
Could you add a test for the `QISKIT_VERSION` and `QISKIT_VERSION_NUMERIC` macros?